airbyte-commons-server/src/main/java/io/airbyte/commons/server/handlers/SchedulerHandler.java

```diff
@@ -472,6 +472,8 @@ private CheckConnectionRead reportConnectionStatus(final SynchronousResponse<Sta
           .status(Enums.convertTo(response.getOutput().getStatus(), StatusEnum.class))
           .message(response.getOutput().getMessage());
     } else {
+      // TODO (Ella): Update logic - 'status' should no longer be necessary, as is the case for the other
+      // two synchronous jobs.
```

Comment:
Creating a separate issue for this change, one I didn't realize was desired during the tech spec, but I think makes sense. Will attach here. TL;DR a lack of output should indicate that the check job itself (not checking the connection) failed.

Reply:
https://app.zenhub.com/workspaces/connector-operations-6318ede5eda32e0010ef8566/issues/gh/airbytehq/airbyte/23451